new/MDDL002.xml

```diff
@@ -0,0 +1,445 @@
+                                <item xml:id="a1">
+                                    <locus target="#2r"></locus>
+                                    <desc type="OwnershipNote"></desc>
+                                    <q xml:lang="gez">አፈ፡ ወርቅ፡ ዘቅዱስ፡<del rend="erasure"></del></q>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus target="#186rb"></locus>
+                                    <desc type="OwnershipNote">The ownership note states that the manuscript is bought by
+                                        <persName ref="PRS2377Bakaffa"/>.</desc>
+                                    <q xml:lang="gez"><hi rend="rubric">ለዝንቱ፡ መጽሐፍ፡ ዘአጥርዮ፡ በን</hi>ዋዮ፡ ንጉሠ፡ ነገሥት፡ ንጉሥ<gap reason="lost"/><persName ref="PRS2377Bakaffa"><hi rend="rubric">በካፋ</hi></persName> ዘሰረቆ ወፈሐቆ በሥልጣነ 
+                                    ጴጥሮስ፡ ወጳውሎስ፡ ውጉዘ፡ ለይኩን፡ ወበረከቶሙ፡ ለቅዱስ፡ ዮሐንስ፡ አፉሁ፡ ዘወርቅ፡ ወለቅዱስ፡ ቄርሎስ፡ ሊቅ፡ የሀሉ፡ ምስለ፡
+                                    ወልዶሙ፡ በጸጋ፡ ሃይማኖት፡ ንጉሥነ <persName ref="PRS2377Bakaffa"><hi rend="rubric">በካፋ</hi></persName> ለዓለም ዓለም አሜን</q>
+                                </item>
+                                <item xml:id="e3">
+                                    <desc>The manuscript contains many corrections throughout (black and red lines above erasures; 
+                                        corrections above erasures, sometimes in condensed script; sometimes dots in the margin next 
+                                        to the lines in which the correction is carried out.)
+                                   </desc>
+                                </item>
+                                <item xml:id="e4">
+                                    <desc><locus from="125vc" to="145vc"/>, in <ref target="#ms_i2"/>, 
+                                        <foreign xml:lang="gez">ዘቄ</foreign>
+                                        <foreign xml:lang="gez">እም</foreign>
+                                        <foreign xml:lang="gez">እ</foreign> were written in the margin.
+                                        These might be an abbreviations standing for  <foreign xml:lang="gez">ዘቄርሎስ</foreign> and
+                                        <foreign xml:lang="gez">እም</foreign>introduces biblical quotations.
+                                   </desc>
+                                </item>
+                                <item xml:id="e5">
+                                    <desc><locus target="#166r"/> The title  <foreign xml:lang="gez">ተረፈ፡ ቄርሎስ፡</foreign> written in the 
+                                        upper margin.
+                                    </desc>
+                                </item>
+                                <item xml:id="e6">
+                                    <desc><locus target="#4rc"/> scribbles in the margin.
+                                    </desc>
+                                </item>
```

Comment:
maybe the title on the mahdar (አፈወርቅ፡) should also be listed as an e7 or in the binding description